index.html

```diff
@@ -10,30 +10,38 @@
       @import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@200;300;400;500;600;700&family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,700;1,800&family=Roboto:ital,wght@0,300;0,400;0,500;0,700;0,900;1,300;1,400;1,500;1,700&display=swap');
       </style>
     <link rel="stylesheet" href="query.css" />
+    <script src="js/script.js" defer></script>
     <title>My Portfolio</title>
 </head>
 <body>
     <header class="header">
-        <a class="logo" href="#">Rubanza Mark</a>
-          <a class="menu" href="#"
-            ><img src="img/IconMenu.png" alt="burger button"
-          /></a>
-          <nav class="topMenu d-flex flex-center">
-            <ul class="d-flex">
-                <li>
-                    <a href="#">Portfolio</a>
-                </li>
-                <li>
-                    <a href="#">About</a>    
-                </li>
-                <li>
-                    <a href="#">Contact</a>                
-                </li>
-            </ul>
-            <a href="mailto:markrubanza10@gmail.com" class="mailIcon">
-                <img src="img/Icon - Mail.png" alt="mailIcon">
-            </a>
-      </header>
+      <a class="logo" href="#">Rubanza Mark</a>
+      <nav class="topMenu d-flex flex-center">
+        <ul class="d-flex">
+          <li>
+            <a href="#">Portfolio</a>
+          </li>
+          <li>
+            <a href="#">About</a>
+          </li>
+          <li>
+            <a href="#">Contact</a>
+          </li>
+        </ul>
+        <a href="mailto:markrubanza10@gmail.com" class="mailIcon">
+          <img src="img/Icon - Mail.png" alt="mailIcon">
+        </a>
+      </nav>
+      <div onclick="toggle(this)">
+        <div class="peakMobile">
+          <a href="#"><img class="imageHamburguer" src="img/IconMenu.png" alt="burger button" /></a>
+          <ul class="newMobile">
+            <li id="portfolio"><a href="#portfolio-section">Portfolio</a></li>
+            <li id="about"><a href="#about-section">About</a></li>
+            <li id="contact"><a href="#contact-section">Contact</a></li>
+          </ul>
+        </div> 
+    </header>
       <main>
         <section class="mainContainer">
           <div class="informativeText">
```

Comment:
- [ ] Kindly fix the design of all section because they are breaking. Keep coding 💪 

Here you have screenshot of your sections:

![image](https://user-images.githubusercontent.com/74506933/157294633-f0b0f805-4526-43b4-ad57-b6ec302d6e17.png)

![image](https://user-images.githubusercontent.com/74506933/157294541-a5b81f33-69fa-40b1-92e8-dab66e318ae6.png)

![image](https://user-images.githubusercontent.com/74506933/157294577-ec6658cb-4ec9-4f7a-b8e0-ee4b88ec54b0.png)